### PR TITLE
[ᚬrc/v0.14.0] Backport windows support and sentry cleanup to v0.14.0

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

File: miner/src/client.rs [hunk 1]
```diff
@@ -149,10 +149,6 @@ impl Client {
                 }
                 Err(e) => {
                     error!("rpc call submit_block error: {:?}", e);
-                    sentry::capture_message(
```
Comment (xxuejie) [inline miner/src/client.rs]: Is this needed for this PR?

Comment (doitian) [reply miner/src/client.rs]: Yes, it can reduce noises in Sentry.

Comment (doitian) [reply miner/src/client.rs]: Not a part of Windows support, just port together.

Comment (doitian) [reply miner/src/client.rs]: Changed PR title

**Pull request merged**